modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
+
+[role="_abstract"]
+When upgrading to a version >= 4.6 from a version < 4.6, you must patch the sensor and admission-control deployments  to set the `POD_NAMESPACE` environment variable.
```

Comment:
I can't access that ibm page without an account, but I'll change this comment 👍

Reply:
Oh, sorry! I think your Red Hat ID (name@redhat.com) should work - it's supposed to take you to SSO (I think). I just am used to providing links in case anyone wonders why I suggested a certain thing - it's not fun reading though :D